enterprise_access/apps/api/serializers/provisioning.py

```diff
@@ -210,3 +210,30 @@ class ProvisioningResponseSerializer(BaseSerializer):
     enterprise_catalog = EnterpriseCatalogResponseSerializer()
     customer_agreement = CustomerAgreementResponseSerializer()
     subscription_plan = SubscriptionPlanResponseSerializer()
+
+
+class SubscriptionPlanOLIUpdateSerializer(BaseSerializer):
+    """
+    Request serializer for updating a SubscriptionPlan's Salesforce OLI.
+    """
+    checkout_intent_id = serializers.UUIDField(
+        help_text='The UUID of the CheckoutIntent associated with this subscription.'
+    )
```

Comment:
[curious] Should we be explicit in calling by renaming this to `checkout-_intent_uuid` given the checkout intent object does contain an `id` attribute already? It may clear up confusion down the line of whether a `checkout_intent_id` references the objects `uuid` or `id` attribute.